docs/data-sources/lb_balancer.md

```diff
@@ -0,0 +1,53 @@
+---
+subcategory: "Elastic Load Balance (ELB)"
+---
+
+# huaweicloud\_lb\_loadbalancer
+
+Use this data source to get available HuaweiCloud elb loadbalancer.
+
+## Example Usage
+
+```hcl
+variable "lb_name" {}
+
+data "huaweicloud_lb_loadbalancer" "test" {
+  name = var.lb_name
+}
+```
+
+## Argument Reference
+
+* `region` - (Optional, String) Specifies the region in which to obtain the load balancer.
+  If omitted, the provider-level region will be used.
+
+* `name` - (Optional, String) Specifies the name of the load balancer.
+
+* `id` - (Optional, String) Specifies the data source ID in UUID format of the load balancer.
+
+* `status` - (Optional, String) Specifies the operating status of the load balancer.
+  Valid values are *ONLINE* and *FROZEN*.
+
+* `description` - (Optional, String) Specifies the supplementary information about the load balancer.
+
+* `vip_address` - (Optional, String) Specifies the private IP address of the load balancer.
+
+* `vip_subnet_id` - (Optional, String) Specifies the ID of the subnet where the load balancer works.
+
+* `enterprise_project_id` - (Optional, String) Specifies the enterprise project id of the load balancer.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `admin_state_up` - The administrative status of the load balancer.
```

Comment:
suggest to deprecate `admin_state_up`, `loadbalancer_provider`,  `security_group_ids` and `tenant_id`